### DISCUSSION TITLE
LEDs illuminate on black or very dark scenes

### DISCUSSION THREAD

**Discussion by willyp713** [awawa-dev/HyperHDR#288] [Q&A]
When I'm watching something that has a very dark scene (like a lot of sci-fi shows and movies) my LEDs will quickly flash to their 'default' scene, which has them all light up in yellow.

So I'll be watching a show and the LEDs work perfectly fine 90% of the time, until we get a space scene, or a dark scene in a horror movie, or whatever, and then it blasts the room with bright yellow light until a less-dark scene comes up and the LEDs return to normal behavior.

It's almost like WLED is losing the signal from HyperHDR and momentarily returns to its default.

Any ideas why this is happening?

**Comment by awawa-dev** [comment]
I do not consider Wifi WLED to be the right solution for HyperHDR ambient lighting systems where a steady stream of data is required to receive with zero or minimal latency for a typical session that lasts several hours. Also, your problem cannot be debugged (so it is not supported) because once UDP is established it almost never throws any error that can be caught by HyperHDR. Perhaps WLED loses connection and reverts to its default yellow color, but since HyperHDR v17 sends a "live" parameter to work around such transient typical WLED problems, it should rather freeze on the last color until communication is restored. So more likely the ESP device is resetting.

**Reply by willyp713** [reply]
Thanks for the detailed reply. As another test, I disabled `Automatic signal detection` and I haven't noticed the issue. So I suspect maybe I just configured that setting incorrectly.

**Reply by awawa-dev** [reply]
If the signal detection was calibrated for a very dark screen then it could happen. You can increase time to trigger the no-signal event (Sleep time is 5 seconds on default). Or set the black color as default in WLED. So even if the no signal is triggered and the connection is lost, WLED should not turn on some bright colors again (but it may take time to establish connection again: Wake time in the automatic detection).

**Comment by awawa-dev** [comment]
Also you may need to activate continues output in the smoothing if you are using udpraw instead of WLED driver for some reason (or HyperHDR 16 or below).
